practice/src/css/components/form.css

```diff
@@ -0,0 +1,42 @@
+.form-wrapper {
+  border-radius: 16px 16px 0;
+  width: 445px;
+  padding: 40px 50px 30px 50px;
+  background-color: var(--primary-background-color);
+}
+
+.form-heading-feature {
+  padding-bottom: 24px;
+  border-bottom: 1px solid var(--tertiary-border-color);
+  width: 70%;
```

Comment:
No need set width here

Reply:
![image](https://github.com/JennyDoJD/javascript-training/assets/110828723/b434551a-c340-469e-9de3-5e75e0e755fc)
I want the bottom border to be like in Figma so I set the width here